src/caselawclient/Client.py

```diff
@@ -38,6 +41,24 @@
 DEFAULT_XSL_TRANSFORM = "accessible-html.xsl"
 
 
+@dataclass
+class SearchParameters:
```

Comment:
Decided to create `SearchParameters` dataclass so that:

- we dont have to specify all kwargs all the way up the search chain
- in turn we dont have to make sure we keep the correct default values of kwargs up the chain - just define it in one place here

Reply:
I've never had to use `@dataclass` in anger but have been looking for an excuse - nice!